apps/fulltextsearch.sh

```diff
@@ -87,51 +89,167 @@ fi
 
 # Check & install docker
 install_docker
+install_docker_compose
```

Comment:
Would it be possible to do it without requiring docker compose or is that not feasible anymore?

Reply:
what I meant was if it is not possible to simply use a docker run command for the container instead of relying on docker-compose?

Reply:
we are not running a vanilla image, there are some addons installed, that will require a third party to host and maintain that new image, with docker-compose there is no need to do that.

Reply:
okay, that is a good reason. Thanks for making that clear!